### DISCUSSION TITLE
[Enhancement] [time-picker] el-time-picker is-range 模式下结束时间无法大于开始时间

### DISCUSSION THREAD

**Discussion by AnkerJenney** [element-plus/element-plus#13816] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `time-picker`

## Description

el-time-picker is-range 模式下结束时间无法大于开始时间 

### Additional comments

期望可以有一个选择任意时间范围的功能

**Comment by chenxch** [comment]
时间区间哪里有开始时间大于结束时间的，如果你有这种场景我认为不适用range，而是用2个时间选择控件实现。

**Reply by JiMuSAMA** [reply]
但是起码可以不让开始时间被结束时间所约束，可以让开始时间不被约束后，选择开始时间结束时间自动向后移，而不是直接禁用掉开始时间的选择能力。
